crm_rma_prodlot_invoice/models/sale_order.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright 2015 Vauxoo
+#    Author: Yanina Aular, Osval Reyes
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import api, models
+
+
+class SaleOrder(models.Model):
+
+    _inherit = 'sale.order'
+
+    @api.one
+    @api.multi
+    def action_ship_create(self):
+        """
+        Set stock moves to invoice lines that matches with
+        product and product quantity
+        """
+        res = super(SaleOrder, self).action_ship_create()
+        move_id = [move for move in self.picking_ids.move_lines]
```

Comment:
@osvalr , aquí tienes un error, checa el issue que levante, para cuando se generan las Delivery orders en mas de un paso, (Shiping and picking), 
https://github.com/Vauxoo/rma/issues/45
https://github.com/Vauxoo/rma/pull/48

Tambien retira @api.one, debido a:
 https://github.com/odoo/odoo/pull/8527
Mas detalles Con @JulioSerna.

Saludos